src/evergreen_names.rs

```diff
@@ -63,3 +65,7 @@ pub const TASK_NAME: &str = "task";
 pub const IDLE_TIMEOUT: &str = "timeout_secs";
 /// Multiversion version combination being run against.
 pub const MULTIVERSION_EXCLUDE_TAGS: &str = "multiversion_exclude_tags_version";
+
+// Build Variant expansions.
+/// Name of large distro for build variant.
+pub const LARGE_DISTRO_NAME: &str = "large_distro_name";
```

Comment:
Can you change this constant name to something like `LARGE_DISTRO_EXPANSION_NAME` or `LARGE_DISTRO_EXPANSION`? The current name confused me for a second and I thought this was the name of the distro in Evergreen.

Reply:
done.